charts/cloudzero-agent/README.md

```diff
@@ -53,7 +75,7 @@ Alternatively, [install the cert-manager CRDs directly](https://cert-manager.io/
 ```console
 kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.16.2/cert-manager.yaml
 ```
-Alternatively, [install the cert-manager CRDs directly](https://cert-manager.io/docs/installation/helm/).
+
```

Comment:
Removed this - I got confused by this section, maybe we need it? @dmepham 

Maybe we can just improve on the layout?